tests/kernel/mem_protect/syscalls/src/main.c

```diff
@@ -15,7 +15,8 @@
 #define SLEEP_MS_LONG	15000
 
 #if defined(CONFIG_BOARD_NUCLEO_F429ZI) || defined(CONFIG_BOARD_NUCLEO_F207ZG) \
-	|| defined(CONFIG_BOARD_NUCLEO_L073RZ)
+	|| defined(CONFIG_BOARD_NUCLEO_L073RZ) \
```

Comment:
I wonder if we should have the SoC, here, not the Board.

Reply:
It would make sense to have the SoC here for sure, but it appears that convention and existing code has boards here.  I had not included this at first, but the build fails without it.